components/serializer.rst

```diff
@@ -528,6 +528,10 @@ There are several types of normalizers available:
     :phpclass:`DateTime` and :phpclass:`DateTimeImmutable`) into strings.
     By default it uses the RFC3339_ format.
 
+.. versionadded:: 3.2
+    Support for specifying datetime format during denormalization was added to
+    ``DateTimeNormalizer`` in Symfony 3.2
```

Comment:
please indent this complete directive to the same level as the description at line 527 - 529.